src/chapters/sql-part1/joins.rst

```diff
@@ -0,0 +1,111 @@
+Joins
+=====
+
+.. index:: ! join
+
+A **join** combines two tables into one result set.
+We can use joins when wanting to query two tables at the same time. 
+Whenever we join two tables, we have to specify the condition upon which the tables need to be joined.
+
+In SQL, there are four different types of joins:
+
+#. Inner Join
+#. Left Outer Join
+#. Right Outer Join
+#. Full Outer Join
+
+.. index:: ! inner join
+
+Inner Join
+----------
+
+Joining two tables on an inner join produces a result set that only includes the values that match in both tables.
+
+.. TODO: Add a diagram to demonstrate what result set looks like.
+
+.. admonition:: Example
+
+   If we use an inner join to combine ``JohnsonWedding`` and ``JohnsonVowRenewal`` in a query, we can see what guests are going to both the vow renewal and the wedding.
+
+   .. sourcecode:: mysql
+
+      SELECT LastName, FirstName
+      FROM JohnsonVowRenewal
+      INNER JOIN JohnsonWedding ON JohnsonVowRenewal.GuestID == JohnsonWedding.GuestID;
+
+   This query will give us a result set of the first and last names of the guests from the ``JohnsonVowRenewal`` table that are also in the ``JohnsonWedding`` table.
```

Comment:
In addition to the diagram and this example, can we add a part that contains specific data?  Maybe something like a recipe table of ingredients vs. a table of pantry contents?  Then the inner join lists the items that are in both tables.

To keep with the wedding theme, maybe join a specific guest table (SHORT) with a specific meal table. The inner join lists guest names and menu items that are both labeled as "vegan".

Reply:
Ok!